### PR TITLE
Update meta to match https://github.com/ws-skeleton releases

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Update meta to match https://github.com/ws-skeleton releases

### Reference 
https://github.com/eclipse/che/issues/10615

### CODE REVIEW

File: plugins/che-dummy-plugin/0.0.1/meta.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-title: Che Java Plugin
-description: ...
-icon: https://....
-url: https://.... # this points to the URL of the tar.gz of the plugin (usually this is the plugin release URL on github)
+type: Che Plugin
+name: Che dummy plugin
+title: Che dummy Plugin
+description: A hello world theia plug-in wrapped into a Che Plug-in
+icon: https://www.eclipse.org/che/images/ico/16x16.png
+url: https://github.com/ws-skeleton/che-dummy-plugin/releases/download/untagged-8d788fa8defd39f5ffd7/che-dummy-plugin.tar.gz
```
Comment (benoitf) [inline plugins/che-dummy-plugin/0.0.1/meta.yaml]: do we have a way to specify "latest" ?
for example https://github.com/ws-skeleton/che-dummy-plugin/releases/latest
is redirecting me to https://github.com/ws-skeleton/che-dummy-plugin/releases/tag/untagged-8d788fa8defd39f5ffd7

Comment (benoitf) [reply plugins/che-dummy-plugin/0.0.1/meta.yaml]: or how to automate updating entries there each time a plug-in is released

Comment (skabashnyuk) [reply plugins/che-dummy-plugin/0.0.1/meta.yaml]: @garagatyi thoughts?

Comment (benoitf) [reply plugins/che-dummy-plugin/0.0.1/meta.yaml]: no, I thought I had accepted it

**Comment by skabashnyuk** [comment]
cool. Thx

**Pull request merged**